contents/handbook/people/onboarding.md

```diff
@@ -51,15 +51,44 @@ Every new joiner at PostHog has an onboarding buddy. If possible, a new joiner w
 - Please make sure you spend at least 3 days together, working through the [first week onboarding list](/handbook/people/onboarding#during-their-first-week) and spending time working on any role-specific tasks that are outlined in the new joiner's personal onboarding issue. 
 - You will remain the new joiner's main point of contact for the first few weeks, so please continue to check in with them at least once a week for the first month or so. 
 
-## In-person Onboarding 
+## In-person onboarding
 
-Occasionally, a new joiner will meet with members of their team in-person to go through the onboarding process. Upon acceptance of an offer, your Team Lead will notify the People & Operations team who will help you coordinate travel if necessary.
+Usually a new joiner will meet with members of their team in-person to go through the onboarding process. Upon acceptance of an offer, your Team Lead will notify the People & Operations team who will help you coordinate travel if necessary.
 
 In these cases, the process is:
+
   - Preemptively create the new team member a Google account
   - Issue them a Brex card to their work email with a sufficiently high temporary balance to cover travel costs
   - Have the new team member book travel as usual
 
+### Engineering
+
+We've made dozens of software engineering hires so far at PostHog. This means we know a thing or two about onboarding engineers by now.  
+We got it down to **four rules for successful onboarding of an engineer**:
+
+1. _On day one_: You got their development environment all set up and shipped something useful together – even if tiny! It feels great to hit the ground running.
+2. _By day two_: You've run at least one 1:1 learning session with the new teammate. Give them all the context they need to succeed. (And by the end of the onboarding, each team member should've run at least one such session.)
```

Comment:
I'd say more is better ... I'd rather say run 1 per day minimum

Reply:
Fair point, it seems overly prescriptive to say "by day two". Reworded the points to avoid that, using "every day" instead in this point.